checksum/dpu/build_configuration.script

```diff
@@ -1,16 +1,5 @@
 //System configuration file
 
-//Create a system with 16 tasklets (0 to 15) to compute individual checksums
-//Stack_size=medium (512 bytes)
-//entry_point=task_main (for each of the 16 tasklets)
-//mailbox_size=2 (Two 32-bits word per tasklet)
-
-tasklet add 0..15 medium task_main 2
-
-// Post the input file size into the system mailbox (One 32-bits word), accessible to any thread
-
-sysmbox 1
-
 // DO NOT EDIT BELOW THIS POINT
 
 save $DIRECTORY/$NAME.json
```

Comment:
We should be able to delete this file completely.

Reply:
Yes, sure. fixed.